projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.pipes.ts

```diff
@@ -89,10 +89,14 @@ export class IgxGridHierarchicalRowPinning implements PipeTransform {
 
     constructor(private gridAPI: GridBaseAPIService<IgxHierarchicalGridComponent>) { }
 
-    public transform(collection: any[], pinnedArea: boolean, pipeTrigger: number): any[] {
+    public transform(collection: any[], isPinned: boolean, pipeTrigger: number): any[] {
```

Comment:
As base grid pinning pipe and the one for the hierarchical grid are the same, we can remove the one for hierarchical grid pinning and re-use the base grid one.